doc/RecordingControl.xml

```diff
@@ -1684,8 +1766,220 @@ SetRecordingJobMode(JobToken, Active)
+      <para>
+        The device shall upload each segment as one object to the object storage.
+        Each object and corresponding segment is identified by its object key.
+        The device shall generate the object key for a segment according to the following ABNF <literal>key</literal> rule (according to RFC 5234):
+      </para>
+      <programlisting>key      = lead "/" time "." counter "." extension
+lead     = [prefix "/"] date ["/" postfix]
+counter  = 1*DIGIT
+date     = fullyear "-" month "-" mday
+time     = hour "-" minute "-" second [secfrac] "Z"
+fullyear = 4DIGIT
+month    = 2DIGIT ; 01-12
+mday     = 2DIGIT ; 01-31
+hour     = 2DIGIT ; 00-23
+minute   = 2DIGIT ; 00-59
+second   = 2DIGIT ; 00-60 (due to leap second rules)
+secfrac  = "." 1*DIGIT</programlisting>
+      <para>
+        If a <literal>Prefix</literal> is configured, the device shall include <literal>prefix</literal> in <literal>lead</literal> and replace it with the configured <literal>Prefix</literal>.
+        If a <literal>Postfix</literal> is configured, the device shall include <literal>postfix</literal> in <literal>lead</literal> and replace it with the configured <literal>Postfix</literal>.
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the segment start in Universal Coordinated Time (UTC), following the ABNF rules.
+        The device shall replace <literal>counter</literal> with a counter reset to 0 for a new span and incremented by 1 when closing segments of the underlying span.
+        The device shall replace <literal>extension</literal> with a value defined by the recording formats in the following sections.
+      </para>
+      <para>
+        Note: <literal>date</literal> is conformant to the extended date format defined in ISO 8601-1.
+        <literal>time</literal> is not conformant to the extended time format defined in ISO 8601-1, as it replaces <literal>":"</literal> with <literal>"-"</literal> compared to that format.
+      </para>
+    </section>
+    <section>
+      <title>Span end objects</title>
+      <para>
+        To store the end time of of a span, the device should store an additional empty object after closing a span.
+        The device shall generate the object key for this object according to the ABNF <literal>key</literal> rule and replacement rules defined in <xref linkend="_refCloudRecordingSegmentObjects" />.
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the span end in Universal Coordinated Time (UTC), following the ABNF rules.
+        The device shall replace <literal>counter</literal> by a value equal to the counter of the last segments of the span plus 1.
+        The device shall replace <literal>extension</literal> with a value defined by the recording formats in the following sections.
+      </para>
+    </section>
+    <section>
```

Comment:
I am not certain this should be included in the specification. It leaves open the blob storage to abuse from the device and increase operation costs for the owner of the cloud subscription. Also, this could create collision if we want to add other files in the future.

Reply:
Do you think we should prohibit the device from uploading any objects under the configured prefix instead?

Reply:
What is the best approach here: remove it from the spec and not mention anything about it or be restrictive and prevent objects other that what is defined in the spec?